book/operators.md

```diff
@@ -123,171 +123,43 @@ Operators are usually case-sensitive when operating on strings. There are a few
 
 ## Spread operator
 
-Nushell has a spread operator (`...`) for unpacking lists and records. You may be familiar with it
-if you've used JavaScript before. Some languages use `*` for their spread/splat operator. It can
-expand lists or records in places where multiple values or key-value pairs are expected.
+The spread operator (`...`) allows to expand lists and records.
 
-There are three places you can use the spread operator:
+lists in a list literal or records in a record literal, as well as expand a list into multiple arguments in a command call.
 
-- [In list literals](#in-list-literals)
-- [In record literals](#in-record-literals)
-- [In command calls](#in-command-calls)
+### Creating lists and records
 
-### In list literals
+When creating a list literal, the spread operator can expand the values of another list. For example, to insert a list into another list:
 
-Suppose you have multiple lists you want to concatenate together, but you also want to intersperse
```

Comment:
I'm not a fan of removing this huge section here. I'd rather have more examples than less. It's nice to be brief and concise but if it doesn't explain the nuances of what it does and how it works, it's not necessarily better.